services/src/main/java/org/keycloak/services/resources/admin/UserResource.java

```diff
@@ -195,6 +195,7 @@ public static void updateUserFromRep(UserModel user, UserRepresentation rep, Set
             user.setUsername(rep.getUsername());
         }
         if (rep.getEmail() != null) user.setEmail(rep.getEmail());
```

Comment:
What about using `StringUtils.isBlank()` here? Then, you do not need these two IFs to check the same variable.
I mean something like this:
```
if (StringUtils.isNotBlank(rep.getEmail())){
    user.setEmail(rep.getEmail());
}else{
    user.setEmail(null);
}


```

Reply:
I think your way is harder to understand.  Possibly just a matter of taste.

Reply:
Thanks for the feedback. Would you mind sharing with us why it is hard to understand?

For me, I would not say that the way coded right now is harder, but for sure, it is unusual. It is executing basically the same check twice.